File: 솝트 알고리즘 스터디/소수찾기.py
def solution(n):
    primes = [False, False] + [True] * (n-1)
    count = 0
    for i in range(2, n+1):
        if primes[i]:
            count += 1
            primes[i*2::i] = [False] * len(primes[i*2::i])
    return count

File: 솝트 알고리즘 스터디/소수찾기.js
function solution(n) {
    let count = 0
    let primes = [false, false]
    for (let i = 2; i <= n; i++) {
        primes.push(true)
    }
    for (let j = 2; j <= n; j++) {
        if (primes[j]) {
            count += 1
            for (let k = 2 * j; k <= n + 1; k += j) {
                primes[k] = false
            }
        }
    }
    return count;
}